module_3/src/com/epam/stringasstringorstringbuilder/StringModifier.java

```diff
@@ -0,0 +1,43 @@
+package com.epam.stringasstringorstringbuilder;
+
+public class StringModifier {
+
+    // task_02
+    public static String insertTextAfterSpecifiedSubstring(String str, String textToInsert, String specifiedSubstr) {
+        String resultString = "";
+        if (str.contains(specifiedSubstr)) {
+            resultString = str.replace(specifiedSubstr, specifiedSubstr + textToInsert);
+        }
+        return resultString;
+    }
+
+    // task_04
+    public static String concatenateSpecifiedWordFromAnotherWord(String donorWord, String specifiedWord) {
+        String resultStr = "";
+        boolean thereAreNotAllCharacters = false;
```

Comment:
It is better to start boolean variables from "is/are" and don't use negation
```suggestion
        boolean areAllCharacters = true;
```